modules/subchannel/test/tests/problems/discretizations/psbt_implicit.i

```diff
@@ -188,12 +188,35 @@ P_out = 4.923e6 # Pa
 
 [Outputs]
   exodus = true
-  [Temp_Out_MATRIX]
-    type = QuadSubChannelNormalSliceValues
-    variable = T
-    execute_on = final
-    file_base = "Temp_Out_Implicit_Segregated.txt"
-    height = 1.0
+  csv = true
+  # [Temp_Out_MATRIX]
```

Comment:
either delete them or uncomment them
I d delete them. Ideally each test is focused on a single problem and has the exact output to check for regressions